style.css

```
@@ -0,0 +1,419 @@
+/* general */
+
+:root{
+    --black:rgb(0, 0, 0);
+    --white:#ffffff;
+    --grey:rgb(85,85,85);
+    --bone:rgb(225, 223, 221);
+}
+
+*{
+    margin:0;
+    padding:0;
+}
+
+html{
+    scroll-behavior: smooth;
+}
+
+body{
+    font-family: "Poppins", sans-serif;
+    background-color:var(--bone);
+}
+
+p{
+    color:var(--grey)
+}
+
+/* desktop-nav */
+
+nav, .nav-links{
+    display:flex;
+}
+
+nav{
+    justify-content: space-around;
+    align-items: center;
+    height: 10vh;
+}
+
+.nav-links{
+    gap:2rem;
+    list-style:none;
+}
+
+/*a{
+    color:var(--black);
+    text-decoration: none;
+    text-decoration-color: #ffffff;
+}
+
+a:hover{
+    color:var(--grey);
+    text-decoration: underline;
+    text-underline-offset: 0.25rem;
+    text-decoration-color:rgb(181,181,181);
+}*/
+
+/* nav-bar hover animation */
+
+a{
+    position: relative;
+    color: #000;
+    text-decoration: none;
+}
+  
+a:hover{
+    color: #000;
+}
+  
+a::before{
+    content: "";
+    position: absolute;
+    display: block;
+    width: 100%;
+    height: 2px;
+    bottom: 0;
+    left: 0;
+    background-color: #000;
+    transform: scaleX(0);
+    transform-origin: top left;
+    transition: transform 0.5s ease;
+}
+  
+a:hover::before{
+    transform: scaleX(1);
+}
+
+.logo{
+    color:var(--black);
+    font-size: 1.5rem;
+    font-weight: 500;
+}
+
+.logo:hover{
+    cursor:default;
+}
+
+/* hamburger-menu */
+
+#hamburger-nav{
+    display:none;
+}
+
+.hamburger-menu{
+    position:relative;
+    display:inline-block;
+}
+
+.hamburger-icon{
+    display:flex;
+    flex-direction: column;
+    justify-content: space-between;
+    cursor: pointer;
+    height:24px;
+    width: 30px;
+}
+
+.hamburger-icon span{
+    width:100%;
+    height:2px;
+    background-color:var(--black);
+    transition:all 0.5 ease-in-out;
```

0.5 ce? Trebuie să dai și unitatea de măsură. 

```suggestion
    transition:all 0.5s ease-in-out;
```